samples/sample-07/src/test/java/com/example/sample07/Sample07ApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.example.sample07;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class Sample07ApplicationTests {
```

Comment:
Why do we still need this class?
Or why don't do the logic in your new class exactly in this one?
So, we won't need to fight for naming 😄